test/parallel/test-assert.js

```diff
@@ -701,12 +701,16 @@ assert.throws(() => {
   code: 'ERR_ASSERTION',
   message: new RegExp(`^'${'A'.repeat(127)} === ''$`)}));
 
-[1, true, false, '', null, Infinity, Symbol('test')].forEach((input) => {
-  assert.throws(
-    () => new assert.AssertionError(input),
-    common.expectsError({
-      code: 'ERR_INVALID_ARG_TYPE',
-      type: TypeError,
-      message: /^The "options" argument must be of type object$/
-    }));
-});
+{
+  // bad args to AssertionError constructor should throw TypeError
+  const args = [1, true, false, '', null, Infinity, Symbol('test'), undefined];
```

Comment:
If you've went to the trouble of making it more readable, why not call is `badArgs` or `invalidArgs`. I've seen the line is 77chars long, but maybe... Maybe just `invalids` or `bad`?